### ISSUE TITLE
Windows 32bit JDK16 - extended.OpenJDK Triage issue for July Release

### ISSUE THREAD

**Issue by adamfarley** [adoptium/aqa-tests#2832]
**Summary**

There are a number of failures here, so this is a temporary issue to list them all and to assess whether they block release.

**[jdk_vector_2 failures](https://ci.adoptopenjdk.net/job/Test_openjdk16_hs_extended.openjdk_x86-32_windows_testList_2/7/consoleFull)**
TESTS: 
- jdk/incubator/vector/AddTest.java
- jdk/incubator/vector/Byte256VectorLoadStoreTests.java
- jdk/incubator/vector/Byte256VectorTests.java
- jdk/incubator/vector/Byte128VectorLoadStoreTests.java
Etc etc (there's over 70 of them, all inside jdk/incubator/vector)

BUG: 
```
00:19:47  java.lang.AssertionError
00:19:47  	at jdk.incubator.vector/jdk.incubator.vector.AbstractSpecies.<init>(AbstractSpecies.java:78)
00:19:47  	at jdk.incubator.vector/jdk.incubator.vector.FloatVector$FloatSpecies.<init>(FloatVector.java:3362)
00:19:47  	at jdk.incubator.vector/jdk.incubator.vector.FloatVector.<clinit>(FloatVector.java:3634)
00:19:47  	at AddTest.<clinit>(AddTest.java:38)
```
Note: The tests that fail are either this error, or are other errors caused by something very much like it (e.g. ByteVector instead of FloatVector).
Also note: These tests all appear to pass on [windows 64bit](https://ci.adoptopenjdk.net/job/Test_openjdk16_hs_extended.openjdk_x86-64_windows_testList_2/9/testReport/)

Assertion: An incubator module, as defined [here](http://openjdk.java.net/jeps/11) is not a module which is declared stable. I take this to mean that quality or reliability is not implied, and the failure of tests associated with such a module should not be used as evidence for blocking a release.

Conclusion: Ignore.

**Workspace permanence issue**

[This job](https://ci.adoptopenjdk.net/job/Test_openjdk16_hs_extended.openjdk_x86-32_windows_testList_1/7/console) appears to have failed because it could not delete the workspace from a previous run.

Machine: test-packet-win2012r2-x64-1 

Folder: "C:\Users\jenkins\workspace\Test_openjdk16_hs_extended.openjdk_x86-32_windows_testList_1"

Proposed Solution: Raise an infra issue to address, disable the node, and re-run the tests.

Conclusion: Rerun.

**Note: All of the below issues can be found in tests run [here](https://ci.adoptopenjdk.net/job/Test_openjdk16_hs_extended.openjdk_x86-32_windows_testList_0/7/consoleFull).**

**tools/launcher/HelpFlagsTest.java**

```
01:01:17  STDERR:
01:01:17  java.lang.AssertionError: HelpFlagsTest failed:
01:01:17  Tool jaccessinspector-32.exe not covered by this test. Add specification to jdkTools array!
01:01:17  Tool jaccesswalker-32.exe not covered by this test. Add specification to jdkTools array!
01:01:17  
01:01:17  	at HelpFlagsTest.main(HelpFlagsTest.java:374)
01:01:17  	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
01:01:17  	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
01:01:17  	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
01:01:17  	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
01:01:17  	at com.sun.javatest.regtest.agent.MainActionHelper$AgentVMRunnable.run(MainActionHelper.java:298)
01:01:17  	at java.base/java.lang.Thread.run(Thread.java:831)
01:01:17  
01:01:17  JavaTest Message: Test threw exception: java.lang.AssertionError
01:01:17  JavaTest Message: shutting down test
```

Note: Mentioned [here](https://github.com/adoptium/aqa-tests/issues/2382#issuecomment-799548468), though it seems unrelated.

Proposed solution: Raise a new issue and exclude the test.

Conclusion: Non-blocking failure, as an incomplete testcase does not infer anything about the quality of the JDK.

**tools/jpackage/windows/WinUpgradeUUIDTest.java#id1**
Also:
- tools/jpackage/windows/WinShortcutTest.java
- tools/jpackage/windows/WinScriptTest.java
- tools/jpackage/windows/WinPerUserInstallTest.java
And many others. See [here](https://ci.adoptopenjdk.net/job/Test_openjdk16_hs_extended.openjdk_x86-32_windows_testList_0/7/consoleFull).

```
light.exe : error LGHT0217 : Error executing ICE action 'ICE01'. The most common cause of this kind of ICE failure is an incorrectly registered scripting engine. See http://wixtoolset.org/documentation/error217/ for details and how to solve this problem. The following string format was not expected by the external UI message logger: "The Windows Installer Service could not be accessed. This can occur if the Windows Installer is not correctly installed. Contact your support personnel for assistance.".
00:59:02      
```

Proposed solution: Raise an infrastructure issue for investigation into test-ibmcloud-win2012r2-x64-2

Conclusion: Wix setup issue.

**tools/jpackage/share/jdk/jpackage/tests/JLinkOptionsTest.java**

Other tests: 
- tools/jpackage/share/jdk/jpackage/tests/AppVersionTest.java

Symptoms: 
```
00:48:30  [18:47:22.700] TRACE: exec: Execute [E:\workspace\Test_openjdk16_hs_extended.openjdk_x86-32_windows_testList_0\aqa-tests\TKG\output_16269939522004\jdk_tools_2\work\scratch\1\.\test.2db6e95c\output\AppVersionTest\AppVersionTest.exe](1); inherit I/O; in directory [.\test.2db6e95c]...
00:48:30  [18:47:22.855] TRACE: exec: Done. Exit code: 1
```

Conclusion: Unsure. Other jpackage failures showed clear signs of a Wix issue, so I'll see if these pass on a different machine, as above.

**compiler/vectorapi/TestLoopStoreVector.java** 

Includes other compiler/vectorapi/* tests also.

Conclusion: these all have "WARNING: Using incubator modules: jdk.incubator.vector" in them, so I'm identifying them as nonblocking failures for the same reasons as above.

**compiler/c2/TestUnsignedByteCompare1.java**

Summary: Failed due to a hang or severe overrun. 

Notes: The stack trace for the test thread looks like it could be related to [this bug](https://bugs.openjdk.java.net/browse/JDK-8261989). Even though our test run appears to lack a c1 option, and tiered compilation is disabled (so probably not the *exact* same issue), the test thread implied that it got stuck trying to run a String.format method.

Conclusion: Run again, and see if it could be an intermittent overrun.

**Comment by adamfarley** [comment]
Things I've tried:

**Re-run the tests**

Can't. Upstream job's artifacts (the build) have expired.

**Re-run the tests against another copy of the build**

The only other copy of the build on that server is located [here](https://ci.adoptopenjdk.net/job/build-scripts/job/openjdk16-pipeline/432/artifact/target/windows/x86-32/hotspot/OpenJDK16U-jdk_x86-32_windows_hotspot_16.0.2_7.zip)

Attempting to re-run against that build produces a complaint that what we downloaded was not a build. I expect it's an authentication issue, and what we downloaded was just a file containing an error message.

**Re-run the above, using the right credentials**

I know where to put the credentials string (CUSTOMIZED_SDK_URL_CREDENTIAL_ID), but I don't know what the right string is.

However, I do know what the right string is for a restricted-access server. Not good for openness, but at least I'll be able to tell if the issues we're seeing are infra related.

**Re-run extended.openjdk testing for win32 on the restricted-access server.**

Attempting to run these tests elsewhere has produced mixed results. In short, the results are invalid because we reached jdk_tools_2, failed several tests, and saw no output until the overrun-kill 7.5 hours later. So we didn't even get a complete run.

**Find a way to re-run the tests on the normal server**

Pending information on how to do that from Slack. Options include:
- Figure out what the credentials string should be.
- Figure out how to upload the build to somewhere accessible to the test.
